src/AdvancedVI.jl

```diff
@@ -47,6 +55,10 @@ function value_and_gradient!(
     return out
 end
 
+function _prepare_gradient(ad::ADTypes.AbstractADType, f, x, aux)
+    return DI.prepare_gradient(f, ad, x, Constant(aux))
+end
+
```

Comment:
It seems like this is not being used anywhere. Is that correct?

Reply:
You're correct. Please feel free to drop it. Also, to be consistent with DI, perhaps change the argument ordering of `value_and_gradient `.